resources/views/components/form-field.blade.php

```diff
@@ -1,33 +1,41 @@
-<div class="{{ ($inline ?? false) ? 'lg:flex' : '' }} {{ ($fullWidth ?? false) ? '' : '' }} mb-2">
-    @if($label ?? false)
-        <div class="{{ ($fullWidth ?? false) ? '' : (($inline ?? false) ? 'lg:w-36 lg:pr-3 lg:text-right' : '') }} pt-1 whitespace-nowrap {{ $errors && $errors->has($attribute ?? null) ? 'text-danger' : '' }}">
-            <label for="{{ $fieldId ?? $attribute ?? null }}">
-                {{ $label }} {{ !empty($required) ? '*' : '' }}
-            </label>
-        </div>
+@props([
+    'help' => null,
+    'hidden' => null,
+    'icon' => null,
+    'id' => null,
```

Comment:
If no ID is given, should one be autogenerated like the child components?

Reply:
Good idea - just to be safe. Added in https://github.com/RetroAchievements/RAWeb/pull/2247/commits/666665af0c4796773e9ceb420619c82e600c797b